### PR TITLE
[swedish] sv_welcome_links-documentation

### PR DESCRIPTION
**Pull request by vladdeSV**

### CODE REVIEW

**Comment by wilzbach** [comment]
Ping @ random Swedish reviewers - @jacob-carlborg would you be so kind to have a look at this translation, too? :)

**Comment by WebFreak001** [comment]
Ping @Vild

File: public/content/sv/welcome/links-documentation.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Länkar och dokumentation
+
+Efter att ha avslutat rundtuden, kan du kolla in några av de följande länkarna för mer information:
```
Comment (Vild) [inline public/content/sv/welcome/links-documentation.md]: rundturen

File: public/content/sv/welcome/links-documentation.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+**NOTERA:** Följande länkar är på engelska
+
+* [Lära sig D som en nybörjare](http://ddili.org/ders/d.en/index.html)
+* [Lära sig D för en erfaren programmerare](http://wiki.dlang.org/Coming_From) från andra språk
+* [D handledning](https://wiki.dlang.org/Tutorials)
+* [Överblick av Ds unika funktioner](http://dlang.org/overview.html)
+* [Vanliga frågom om D](http://dlang.org/faq.html)
```
Comment (Vild) [inline public/content/sv/welcome/links-documentation.md]: frågor

File: public/content/sv/welcome/links-documentation.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Länkar och dokumentation
+
+Efter att ha avslutat rundtuden, kan du kolla in några av de följande länkarna för mer information:
+
+**NOTERA:** Följande länkar är på engelska
+
+* [Lära sig D som en nybörjare](http://ddili.org/ders/d.en/index.html)
+* [Lära sig D för en erfaren programmerare](http://wiki.dlang.org/Coming_From) från andra språk
```
Comment (jacob-carlborg) [inline public/content/sv/welcome/links-documentation.md]: `Lära sig D` sounds a bit weird to me.

Perhaps:
`Lär dig D, för nybörjare`
`Lär dig D, för erfarna programmerare`

@Vild, what do you think?

Comment (Vild) [inline public/content/sv/welcome/links-documentation.md]: I agree that it sounded a bit strange.
I think your suggestions would be better.

**Comment by vladdeSV** [comment]
Updated according to suggestions. Thanks!

**Comment by wilzbach** [comment]
@vild @jacob-carlborg thanks for your amazingly fast review! Once you have approved the changes, we can ship this ;-)

File: public/content/sv/welcome/links-documentation.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+* [Lär dig D, för nybörjare](http://ddili.org/ders/d.en/index.html)
+* [Lär dig D, för erfarna programmerare](http://wiki.dlang.org/Coming_From) från andra språk
+* [D handledning](https://wiki.dlang.org/Tutorials)
+* [Överblick av Ds unika funktioner](http://dlang.org/overview.html)
+* [Vanliga frågor om D](http://dlang.org/faq.html)
+
+### Hjälp
+
+* Fråga frågor på [#d](irc://irc.freenode.net/d) IRC kanalen på freenode ([webb gränssnitt](https://kiwiirc.com/client/irc.freenode.net/d))
```
Comment (Vild) [inline public/content/sv/welcome/links-documentation.md]: `webb gränssnitt` -> `webbgränssnitt`

File: public/content/sv/welcome/links-documentation.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+* Få hjälp på [D.learn](http://forum.dlang.org/group/learn)
+* Öppna en 'issue' [i D bugg trackern](https://issues.dlang.org)
```
Comment (Vild) [inline public/content/sv/welcome/links-documentation.md]: Something I forgot to comment on before is if `bugg trackern` the best word for the issue tracker.
The closes thing I can find is `ärendehantering system`, but this doesn't really tell you it is about bugs.

Comment (Vild) [inline public/content/sv/welcome/links-documentation.md]: I guess `felspårningssystemet` would be the best word for this.

A translation in KDE translates this: `[...] known issues, then in the bug tracking system. `
Into this: `[...] kända problem, och titta därefter i felspårningssystemet.`

File: public/content/sv/welcome/links-documentation.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Länkar och dokumentation
+
+Efter att ha avslutat rundturen, kan du kolla in några av de följande länkarna för mer information:
+
+**NOTERA:** Följande länkar är på engelska
```
Comment (wilzbach) [inline public/content/sv/welcome/links-documentation.md]: FYI: if there are any reference(s) in Swedish , just add them
(however I assume that there isn't much Swedish content)

**Comment by Vild** [comment]
Everything looks good what I can see

**Comment by wilzbach** [comment]
> Everything looks good what I can see

Thanks again to everyone!!

File: public/content/sv/welcome/links-documentation.md (hunk 5 above)
Comment (jacob-carlborg) [inline public/content/sv/welcome/links-documentation.md]: I don't agree. I can't recall that I have ever heard the word `felspårningssystemet`. In my experience, where I have worked, the most common thing is to use the actual name of the issue tracking system that the company is using, i.e. bugzilla, github, jira and so on. I guess the next common thing is to use an english word. It's pretty common to mix swedish with english words in software development and computer related topics. If you really want to have a swedish word I think `ärendehanteringssystem` is the best.

**Pull request merged**